backend/lib/database/postgres/postgis.py

```diff
@@ -43,6 +42,15 @@ def to_polygon(long_lat_tuples):
 
 
 def install():
+    """Use appropriate setup function for AWS environment or otherwise."""
+    # todo reach consensus on how we would like to do this
+    if os.getenv('IS_AWS') == 'TRUE':
```

Comment:
can you add to README?

Reply:
yeah i will add documentation for what is required for however we decide to do this

Reply:
i think something like having a makefile command to get it to work would be easier. also still not clear on how to add data to local db